terracumber_config/tf_files/SUSEManager-4.1-refenv-PRV.tf

```diff
@@ -106,6 +106,10 @@ module "srv" {
   provider_settings = {
     mac    = "52:54:00:00:00:32"
     memory = 8192
+    additional_repos = {
```

Comment:
this should be placed outside the provider settings

Reply:
You right. Wrong copy paste.